Sources/Swift/Integrations/SessionReplay/SentrySRDefaultBreadcrumbConverter.swift

```diff
@@ -3,7 +3,8 @@ import Foundation
 
 @objc
 protocol SentryReplayBreadcrumbConverter: NSObjectProtocol {
-    func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent]
+    func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEventProtocol]
+    func convert(from breadcrumb: Breadcrumb) -> SentryRRWebEventProtocol?
```

Comment:
Why do we need to define both now? Did you mean to remove the original method and replace it with a single-breadcrumb one? I'd be in favor of that. 
Otherwise, this change makes no sense on the consumer side because `convertFromBreadcrumb` would never be called anyway.

Reply:
Yes. I know this is weird.
For `func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date)` just copy the implementation from `SentrySRDefaultBreadcrumbConverter`

I will fix all of this when I convert `SentrySessionReplay` to Swift. I don't want to deal with this things in Objc right now.

Reply:
in case it helps, here's the objc implementation of the filter:

```objective-c

- (NSArray<SentryRRWebEvent *> *_Nonnull)
    convertWithBreadcrumbs:(NSArray<SentryBreadcrumb *> *_Nonnull)breadcrumbs
                      from:(NSDate *_Nonnull)from
                     until:(NSDate *_Nonnull)until {
  NSMutableArray<SentryRRWebEvent *> *outBreadcrumbs = [NSMutableArray array];
rrwebBreadcrumb;
  for (SentryBreadcrumb *breadcrumb in breadcrumbs) {
    // - (NSComparisonResult)compare:(NSDate *)other;
    // If:
    // The receiver and `other` are exactly equal to each other, NSOrderedSame
    // The receiver is later in time than `other`, NSOrderedDescending
    // The receiver is earlier in time than `other`, NSOrderedAscending.
    if (!breadcrumb.timestamp ||
        [breadcrumb.timestamp compare:from] == NSOrderedAscending ||
        [breadcrumb.timestamp compare:until] == NSOrderedDescending) {
      continue;
    }

    SentryRRWebEvent * rrwebBreadcrumb = [selfconvertFrom:breadcrumb];
    if (rrwebBreadcrumb) {
      [outBreadcrumbs addObject:rrwebBreadcrumb];
    }
  }
  return outBreadcrumbs;
}

Reply:
You won't let this one go, right? 🤣

Reply:
thx :)